src/components/screens/sprint-view-screen.tsx

```diff
@@ -71,7 +68,8 @@ const SprintViewScreen = () => {
           }); 
           return totalMinutes;
         } catch (error) {
```

Comment:
I can't add a comment on the correct line but this is referring to the above timeEntriesApi.listProjectTimeEntries request.

rather than have projectId: allocation.project || 0
You should check if allocation.project is defined on the first line of the map function before the try, and throw the appropriate error if not. This means you will be able to just do projectId: allocation.project. 
Using the || 0 is unclear.